src/main.rs

```diff
@@ -56,34 +59,63 @@ fn main() {
                 // Can this ever happen?
                 Ok(())
             } else {
-                println!("Fuzzing found errors!");
+                if let Some(ref mut terminal) = term_stdout {
+                    let _ = terminal.fg(term::color::YELLOW);
+                    println!("Fuzzing found errors!");
+                    let _ = terminal.reset();
+                } else {
+                    println!("Fuzzing found errors!");
+                }
                 process::exit(-1)
             }
         } else {
             result.map(|_| ())
         }
     } else if args.flag_list {
-        list_fuzz_targets()
+        list_fuzz_targets(&mut term_stdout)
             .map(|_| ())
     } else {
-        println!("Invalid arguments. Usage:\n{}", USAGE);
+        if let Some(ref mut terminal) = term_stdout {
+            let _ = terminal.fg(term::color::GREEN);
+            println!("Invalid arguments. Usage:\n{}", USAGE);
```

Comment:
These need to be written to the term stdout/err (probably stderr)

Also, can we follow the cargo/rustc convention here? They show a white bold "error: some error message" where `error` is red. So we would display one bold/colorful line, followed by the usage in the default color.